creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -409,17 +409,47 @@ spec:
   - name: red
     multus:
       networkName: red
------
+----
 
 At this time, `bridge` mode doesn’t support additional configuration
 fields.
 
 ______________________________________________________________________________
-*Note:* due to IPv4 address delagation, in `bridge` mode the pod doesn’t have
+*Note:* due to IPv4 address delegation, in `bridge` mode the pod doesn’t have
 an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* admin can forbid using `bridge` interface type for `pod networks` via a 
+designated configuration flag. 
+To achieve it, the admin should set the following option to `false`:
+______________________________________________________________________________
+
+
+[source,yaml]
+-----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
```

Comment:
(sorry) do we still use kube-system? I thought we switched to a different one. I suggest you trim this completely from the example like we do for other examples.

Reply:
Thanks @booxter .You are right, the namespace is "kubevirt".
I saw that most of the configMap examples spread across the user-guide do include namespace, even though in this file most of the examples (not configMap examples) don't.
would you prefer to trim namespace or update it to "kubevirt"?

Reply:
Either way, as long as you don't put the obsolete name into examples. Since you may want to clean up other examples by e.g. adding the right namespace name to them, if I were you, I would leave the namespace out of new examples in this PR and then send a follow-up PR that adds correct namespaces to all examples in this file. But that's after this PR lands.

Reply:
The thing is that config-map has to be namespaced under kubevirt namespace so the configMap example should include kubevirt-namespace ,while VMs and NADs can also use the default namespce so the examples can also not include namespace.
what do you think @booxter ?